relay-sampling/src/lib.rs

```diff
@@ -492,6 +477,21 @@ impl SampleRateEvaluator {
     }
 }
 
+/// A sampling rule that has been successfully matched and that contains all the required data
+/// to return the sample rate.
+#[derive(Debug, Clone, Copy)]
+pub struct ActiveRule {
+    pub id: RuleId,
+    evaluator: SampleRateEvaluator,
+}
+
+impl ActiveRule {
+    /// Gets the sample rate for the specific rule.
+    pub fn get_sample_rate(&self, now: DateTime<Utc>) -> f64 {
+        self.evaluator.evaluate(now)
+    }
+}
```

Comment:
nit: Because `SamplingRule` uses active rule, I would change the declaration order to 

```
SampleRateEvaluator
ActiveRule
SamplingRule
```